rsts/deployment/deployment/multicluster.rst

```diff
@@ -24,104 +24,279 @@ Scaling Beyond Kubernetes
+                    "oidc.eks.<REGION>.amazonaws.com/id/<CONTROLPLANE-OIDC-PROVIDER>:sub": [
+                        "system:serviceaccount:flyte:flyteadmin",
+                        "system:serviceaccount:flyte:datacatalog"
+                          ]
+                      }
+                  }
+              }
+          ]
+      }
+
+   **Data plane role**
+
+   1. Create the role and Trust Relationship:
+
+   .. prompt:: bash
+      
+      eksctl create iamserviceaccount --cluster=<dataplane1-cluster-name> --name=flytepropeller --role-only --role-name=flyte-dataplane-role --attach-policy-arn <ARN-of-your-IAM-policy> --approve --region <AWS-REGION-CODE> --namespace flyte
+      
+   2. Edit the **Trust Relationship** of the data plane role
+
+   .. note:: 
+
+      By default, every Pod created for Task execution, uses the ``default`` Service Account on their respective namespace. In your cluster, you'll have as many
+      namespaces as ``project`` and ``domain`` combinations you may have. Hence, it might be useful to use a ``StringLike`` condition and to use a wildcard for the namespace name in the Trust Policy
+
+   3. Add the ``default`` Service Account:
+   
+   
+   Example configuration for one data plane cluster:
+
+   .. code-block:: json
+
+      {
+      "Version": "2012-10-17",
+      "Statement": [
+        {
+            "Effect": "Allow",
+            "Principal": {
+                "Federated": "arn:aws:iam::<AWS-ACCOUNT-ID>:oidc-provider/oidc.eks.<AWS-REGION-CODE>.amazonaws.com/id/<DATAPLANE1-OIDC-PROVIDER>"
+            },
+            "Action": "sts:AssumeRoleWithWebIdentity",
+            "Condition": {
+                "StringLike": {
+                    "oidc.eks.<AWS-REGION-CODE>.amazonaws.com/id/<DATAPLANE1-OIDC-PROVIDER>.:aud": "sts.amazonaws.com",
+                    "oidc.eks.<AWS-REGION-CODE>.amazonaws.com/id/<DATAPLANE1-OIDC-PROVIDER>.:sub": [
+                        "system:serviceaccount:flyte:flytepropeller",
+                        "system:serviceaccount:*:default"
+                    ]
+                }
+            }
+        }
+
+    
+.. _dataplane-deployment:
 
 Data Plane Deployment
 *********************
 
-First, add the Flyteorg Helm repo
+This guide assumes that you have two Kubernetes clusters and that you can access
+them all with ``kubectl``.
+
+Let's call these clusters ``dataplane1`` and ``dataplane2``. In this section, you'll prepare
+the first cluster only. 
 
-.. code-block::
+1. Add the ``flyteorg`` Helm repo:
+
+.. prompt:: bash
 
     helm repo add flyteorg https://flyteorg.github.io/flyte
     helm repo update
     # Get flyte-core helm chart
     helm fetch --untar --untardir . flyteorg/flyte-core
     cd flyte-core
 
-Install Flyte data plane Helm chart
+2. Open the ``values-dataplane.yaml`` file and add the following contents:
+
+   .. code-block:: yaml
+
+      configmap:
```

Comment:
I think you also need the configuration for datacatalog; in my case I had to also add the endpoint for that.
Note that this should be moved to the default charts and not left to the user